src/Framework/Runner/Interfaces.cs

```diff
@@ -199,7 +200,7 @@ public interface IRunner
         /// <summary>
         /// Setup tests. Precedes a call to RunAllTests.
```

Comment:
now that this returns a bool can you update this summary?

Reply:
:+1: